clients/admin-ui/src/features/privacy-requests/RequestDetails.tsx

```diff
@@ -56,7 +58,11 @@ const RequestDetails = ({ subjectRequest }: RequestDetailsProps) => {
             />
           )}
 
-          <DaysLeftTag daysLeft={subjectRequest.days_left} includeText />
+          <DaysLeftTag
+            daysLeft={subjectRequest.days_left}
+            includeText
+            status={subjectRequest.status as ApiPrivacyRequestStatus}
```

Comment:
ah okay I see why you did this—how about we make a ticket to update our custom types with all the API types? would be a 💯 tech debt ticket to do

Reply:
That's great tech debt idea. I'll make that ticket now and add it to the sprint backlog.